<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutPage</class>
 <widget class="QWidget" name="AboutPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>80</number>
   </property>
   <property name="topMargin">
    <number>32</number>
   </property>
   <property name="rightMargin">
    <number>80</number>
   </property>
   <property name="bottomMargin">
    <number>32</number>
   </property>
   <item>
    <widget class="ScrollArea" name="ScrollArea">
     <property name="frameShape">
      <enum>QFrame::Shape::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>619</width>
        <height>608</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="scrollLayout">
       <property name="spacing">
        <number>16</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="headerContainer" native="true">
         <layout class="QHBoxLayout" name="headerLayout">
          <property name="spacing">
           <number>16</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="ImageLabel" name="logoLabel">
            <property name="minimumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="TitleLabel" name="appNameLabel">
            <property name="text">
             <string>Stellar Neo</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="headerSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="SubtitleLabel" name="aboutAppTitle">
         <property name="text">
          <string>關於應用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="BodyLabel" name="aboutAppBody">
         <property name="text">
          <string>鑒於市面上缺乏令人滿意的浮水印應用，特別是在讀取 LRC 或 Luminar Neo 導出的圖片 EXIF 資訊方面存在不足，Stellar Neo 因此誕生。本應用致力於提供真正開箱即用、注重細節優化的相框與浮水印工具，希望能成為大家的首選。</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_1">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="SubtitleLabel" name="aboutAuthorTitle">
         <property name="text">
          <string>關於作者</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="BodyLabel" name="authorNameLabel">
         <property name="text">
          <string>Moon Cancer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="BodyLabel" name="authorBioLabel">
         <property name="text">
          <string>Let's go to the pier and grab some fries. Ciallo～(∠・ω&lt; )⌒☆</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="authorLinksContainer" native="true">
         <layout class="QHBoxLayout" name="authorLinksLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="HyperlinkButton" name="githubButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>GitHub</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="HyperlinkButton" name="blogButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Rem Blog</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="authorLinksSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="SubtitleLabel" name="creditsTitle">
         <property name="text">
          <string>致謝與聲明</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="logoCreditContainer" native="true">
         <layout class="QHBoxLayout" name="logoCreditLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="BodyLabel" name="logoCreditPrefixLabel">
            <property name="text">
             <string>應用 Logo 來源：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="HyperlinkButton" name="logoArtButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>「私」</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="BodyLabel" name="logoCreditByLabel">
            <property name="text">
             <string>by</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="HyperlinkButton" name="logoArtistButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>ｻｻﾞﾅﾐ。</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="logoCreditSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="CaptionLabel" name="disclaimerLabel">
         <property name="text">
          <string>此應用涉及的品牌 Logo 僅用於學習與參考，請勿用於任何商業用途。</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="mainSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrollArea</class>
   <extends>QScrollArea</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SubtitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>BodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CaptionLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>HyperlinkButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>ImageLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
